.droneci/NiceUbuntuDockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:latest
```

Comment:
BTW debian/testing image is usually much smaller and often contains more recent versions of compilers